Source/WebKit/UIProcess/ios/PageClientImplIOS.mm

```diff
@@ -1171,6 +1171,11 @@ static UIInterfaceOrientationMask toUIInterfaceOrientationMask(WebCore::ScreenOr
     return roundAndClampToSRGBALossy([webView() _insertionPointColor].CGColor);
 }
 
+bool PageClientImpl::isScreenBeingCaptured()
+{
+    return [[[contentView() window] screen] isCaptured];
```

Comment:
Nit - you can probably factor this out into a helper property on `WKContentView` and use it here and in `-didMoveToWindow `